app/api/app-revalidate/route.ts

```diff
@@ -24,7 +27,16 @@ export async function GET(request: NextRequest): Promise<NextResponse> {
   }
 
   if (uri) {
-    revalidatePath(`/[locale]/${uri}`, "page");
+    languages.forEach((locale) => {
+      const parts: Array<string> = uri === CRAFT_HOMEPAGE_URI ? [] : [uri];
+      if (locale !== fallbackLng) {
+        parts.unshift(locale);
+      }
+
+      revalidatePath(`/${parts.join("/")}`);
+    });
+
+    revalidateTag(tags.globals);
```

Comment:
revalidate globals on any revalidate so stuff like navigation changing is picked up by all pages